Makefile.Common

```diff
@@ -34,7 +34,7 @@ fmt:
 .PHONY: tidy
 tidy:
 	rm -fr go.sum
-	$(GOCMD) mod tidy -compat=1.17
+	$(GOCMD) mod tidy -compat=1.18
```

Comment:
We cannot do this at this time.  We still need to support building with 1.17.  See our [version support policy](https://github.com/open-telemetry/opentelemetry-collector/blob/main/README.md#compatibility).

Reply:
I see, thanks for pointing that out!